README.rst

```diff
@@ -541,6 +554,11 @@ Choose from the list of available fixers:
                         the class name should match
                         the file name. (Risky fixer!)
 
+* **random_api_migration**
```

Comment:
And then you do not test `rand` -> `mt_rand`, please check the ref with example I provided.